HOLMS.Messaging/RabbitConfiguration.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace HOLMS.Messaging {
+    public class RabbitConfiguration {
+        public string Host { get; protected set; }
+        public ushort Port { get; protected set; }
+        public string User { get; protected set; }
+        public string Password { get; protected set; }
+        public string VHost { get; protected set; }
+
+        public RabbitConfiguration(string connectionString) {
+            //example: amqp://user:pass@hostname:port/vhost
+            int protocolEndIndex = connectionString.IndexOf("://");
```

Comment:
Rather than doing all this low-level indexed string manipulation, which invites error, you might consider an approach that just splitting (tokenizing) the string.

I think connection strings were meant for this, e.g. first split on "://", first part is protocol, then on /, then on "@", then finally on ":". It's a little more work but probably less brittle code. 

If it works as-written, just keep it -- indexes are one of those things I try to avoid, in the name of robustness and readability.